app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -266,8 +269,9 @@ class ParkingViewModel(
     _closestParkings.value =
         _rectParkings.value
             .filter { parking ->
-              TurfMeasurement.distance(_circleCenter.value!!, parking.location.center) *
-                  KM_TO_METERS <= _radius.value
+              TurfMeasurement.distance(
```

Comment:
I am not quite sure why we filter parkings on arrival? By construction, when we add a parking, it shouldn't be greater than the given area, so there shouldn't be any invalid parkings in the database?

Reply:
Umm the only change i did here was the use of a unit parameter to compute the distance in meters in rather than declaring a constant. This is completely unrelated to my task but I took the chance to make the change